src/types.ts

```diff
@@ -457,6 +457,16 @@ export interface TradesRow<BigNumberType> extends BaseTransactionRow {
   tradeGroupId: Bytes32|null;
 }
 
+export interface CompleteSetsRow<BigNumberType> extends BaseTransactionRow {
+  marketId: Address;
+  account: Address;
+  eventName: string;
+  numPurchasedOrSold: BigNumberType;
+  numCompleteSets: BigNumberType;
+  tradeGroupId: Bytes32|null;
+  timestamp: number|null;
```

Comment:
timestamp shouldn't be able to be null? I think this has to do with how you're storing the timestamp on block-processing. if you fix that, timestamp can always be a number